integrationTests/testProcessorNode.go

```diff
@@ -15,6 +15,8 @@ import (
 	"github.com/ElrondNetwork/elrond-go/core"
 	"github.com/ElrondNetwork/elrond-go/core/accumulator"
 	"github.com/ElrondNetwork/elrond-go/core/check"
+	"github.com/ElrondNetwork/elrond-go/core/indexer"
+	coreMock "github.com/ElrondNetwork/elrond-go/core/mock"
```

Comment:
do not import mock from another package.

Reply:
Fixed by moving it to `testscommon`, where we hold another commonly-used mocks.